tests/bluetooth/tester/src/btp_gatt.c

```diff
@@ -2319,6 +2320,64 @@ static uint8_t get_attr_val(const void *cmd, uint16_t cmd_len,
 	return BTP_STATUS_FAILED;
 }
 
+static struct bt_uuid_128 test_uuid = BT_UUID_INIT_128(
+	0x94, 0x99, 0xb6, 0xa9, 0xcd, 0x1c, 0x42, 0x95,
+	0xb2, 0x07, 0x2f, 0x7f, 0xec, 0xc0, 0xc7, 0x5b);
+
+static struct bt_gatt_attr test_attrs[] = {
+	BT_GATT_PRIMARY_SERVICE(&test_uuid),
+};
+
+static struct bt_gatt_service test_service = BT_GATT_SERVICE(test_attrs);
+
+static bool test_service_registered;
```

Comment:
This could be moved into the `change_database` function to reduce its scope

Reply:
actually, I cannot move it to function scope because than those are not constant and I get "error: initializer element is not constant"  when doing BT_GATT_PRIMARY_SERVICE()

Reply:
Not even if you add `const`? Or does the GATT API not like that? 

At least `static bool test_service_registered;` can moved then :)